src/main/java/com/plivo/api/models/base/Lister.java

```diff
@@ -80,18 +80,6 @@ public Integer offset() {
     return response.body();
   }
 
-  public Long get() throws IOException, PlivoRestException {
```

Comment:
Why are we removing this? Are there no other consumers of this base class?

Reply:
@manas-plivo and @narayana-plivo confirmed that this method is only being used in PowerPackLister